src/options/view.rs

```diff
@@ -29,27 +29,26 @@ impl Mode {
     ///
     /// This is complicated a little by the fact that `--grid` and `--tree`
     /// can also combine with `--long`, so care has to be taken to use the
-    pub fn deduce<V: Vars>(matches: &MatchedFlags<'_>, vars: &V) -> Result<Self, OptionsError> {
-        let flag = matches.has_where_any(|f| f.matches(&flags::LONG) || f.matches(&flags::ONE_LINE)
-                                          || f.matches(&flags::GRID) || f.matches(&flags::TREE));
+    pub fn deduce<V: Vars>(matches: &Opts, vars: &V, stricness: bool) -> Result<Self, OptionsError> {
+        let _flag = matches.long > 0 || matches.oneline > 0 || matches.grid > 0 || (matches.tree > 0);
 
-        let Some(flag) = flag else {
-            Self::strict_check_long_flags(matches)?;
+        if ! _flag {
+            Self::strict_check_long_flags(matches, stricness)?;
             let grid = grid::Options::deduce(matches)?;
             return Ok(Self::Grid(grid));
         };
 
-        if flag.matches(&flags::LONG)
-        || (flag.matches(&flags::TREE) && matches.has(&flags::LONG)?)
-        || (flag.matches(&flags::GRID) && matches.has(&flags::LONG)?)
+        if (matches.long > 0 && matches.grid == 0 && (matches.tree == 0)) 
+        || (matches.tree > 0 && matches.long > 0)
+        || (matches.grid > 0 && matches.long > 0)
         {
-            let _ = matches.has(&flags::LONG)?;
+            let _ = matches.long;
             let details = details::Options::deduce_long(matches, vars)?;
 
-            let flag = matches.has_where_any(|f| f.matches(&flags::GRID) || f.matches(&flags::TREE));
+            let flag = matches.grid > 0 || (matches.tree > 0);
 
-            if flag.is_some() && flag.unwrap().matches(&flags::GRID) {
-                let _ = matches.has(&flags::GRID)?;
+            if flag && matches.grid > 0{
+                let _ = matches.grid;
```

Comment:
Does this line does something/is it useful after the migration to clap?

Reply:
I dont know I just letted it like that but it seems to be unuseful because _ in rust means unused